README.md

```diff
@@ -30,6 +31,7 @@ Pages] site.
 This program has an optional build dependencies on the following libraries:
 
 * libncurses5-dev (without it 'switchtec gui' will not work)
+* libssl-dev (without it some of the commands in 'switchtec mfg' will not work)
```

Comment:
Both these lines should have a comma after "without it".

Reply:
Updated.